From 62c7502c19a4e7bb4519abc599c4501a2b0c3854 Mon Sep 17 00:00:00 2001
From: Mathis Ribet <mathis.ribet@2021.icam.fr>
Date: Tue, 14 Jan 2025 20:12:08 +0100
Subject: [PATCH] Fix exit code

int/bool confusion caused the exit code to be inverted
--- a/netbox_agent/cli.py
+++ b/netbox_agent/cli.py
@@ -43,7 +43,7 @@ def run(config):
 
     if version.parse(nb.version) < version.parse("3.7"):
         print("netbox-agent is not compatible with Netbox prior to version 3.7")
-        return False
+        return 1
 
     if (
         config.register
@@ -56,7 +56,7 @@ def run(config):
         server.netbox_create_or_update(config)
     if config.debug:
         server.print_debug()
-    return True
+    return 0
 
 
 def main():
